client/client-multi/client-c/src/platform/system_logger.h

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2014 CyberVision, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef SYSTEM_LOGGER_H_
+#define SYSTEM_LOGGER_H_
+
+#include "../kaa_error.h"
+
+/*
+ * Write log message in system dependent way.
```

Comment:
This documentation is insufficient. Please take a look at the other headers where functions are fully documented in doxygen-compatible fashion, including @brief, @param, and @return at least.